mobileversion.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+     <meta charset="UTF-8" >
+     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+     <link rel="stylesheet" href="mobileversion.css">
+     <link rel="preconnect" href="https://fonts.googleapis.com">
+     <link rel="preconnect" href="https://fonts.googleapis.com">
+     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+     <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+     <title>Mobile Version Website</title>
+  </head>
+    <body>
+       <header class="header">
+        <nav>
+     <a href="#">My Logo</a>
+     <a href="#"><i class="fa fa-bars"></i></a>
+     </nav>
+        </header>
+             <section>
+                <div class="flex-container">
+                    <div class="item5">
+            </div>
+            <div class="item6">
+              <div>
+                <h1 class="mariane">I'm Mariane<br>
+                  Glad to see<br>
+                 you!<br>
+               </h1>
+              </div>
+                <div>
+                <p class="text">
+                  I’m a software developer! I can help you<br>
+                   build a product , feature or website Look <br>
+                   through some of my work and<br>
+                    experience! If you like what you see and <br>
+                    have a project you need coded, don’t<br>
+                     hestiate to contact me.
+                </p>
+              </div>
+              <div >
+                <h2 class="cap">LET’S CONNECT</h2>
+                <ul class="lass">
+                  <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+                  <li><a href="#"><i class='fab fa-linkedin-in'></i></a></li>
+                  <li><a href="#"><i class='fab fa-medium-m'></i></a></li>
+                  <li><a href="#"><i class='fab fa-github'></i></a></li> 
+                  <li><a href="#"><i class='fab fa-angellist'></i></a></li> 
+                </ul>
+              </div>
+              </div>
+           </div>
+        </section>
+        <section>
```

Comment:
[OPTIONAL] I suggest you use `display:grid` for this section tag. It is a [core requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)
<img width="1667" alt="Screenshot 2022-10-29 at 15 11 50" src="https://user-images.githubusercontent.com/48546992/198830814-7087af4a-2862-405b-bef8-68c6ab6e4ccc.png">